docs/HACKING.md

```diff
@@ -163,90 +258,149 @@ If the bundle channel needs to be changed, the `CHANNELS` and/or `DEFAULT_CHANNE
 ```
+
 The variables in the command should be changed to match the container image repository you wish to store the bundle in.
 You can also change the channels based on the release status of the operator.
-This command should create a new bundle image. Bundle image and operator image are two different images. 
+This command should create a new bundle image. Bundle image and operator image are two different images.
 
 You should then push the newly created bundle image to the remote repository:
+
 ```shell script
 podman push $BUNDLE_REPOSITORY:$BUNDLE_TAG
 ```
 
 You should verify that the new bundle is valid:
+
 ```shell script
 operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_TAG --image-builder podman
 ```
 
-### Creating a new operator index
+### Create a CatalogSource using the index
 
-#### Pre-requisites
-[opm](https://github.com/operator-framework/operator-registry/) has been installed on the build system.
-All previous [pre-requisites](#pre-requisites) must be satisfied as well.
+See the OpenShift docs found
+[here](https://docs.openshift.com/container-platform/latest/operators/admin/olm-managing-custom-catalogs.html#olm-creating-catalog-from-index_olm-managing-custom-catalogs)
 
-You can skip this step if you want to run the operator for [developer testing purposes only](#development-workflow)
+### Follow the normal procedure to deploy an operator
 
-An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
-cluster.
+See the OpenShift docs found
+[here](https://docs.openshift.com/container-platform/latest/operators/admin/olm-adding-operators-to-cluster.html)
+
+## Test
+
+If you are not running e2e tests, it is likely that you will want to deploy a custom workload to test your changes with.
+Find more information [here](custom-workload.md)
+
+## Cleanup
+
+### Cleaning up a manual deployment  
+
+To remove the installed resources:
 
 ```shell script
-opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$INDEX_TAG --container-tool podman
+make undeploy
 ```
 
-You should then push the newly created index image to the remote repository:
+## Cleaning up an OLM deployment
+
 ```shell script
-podman push $INDEX_REPOSITORY:$INDEX_TAG
+hack/olm.sh cleanup
 ```
 
-#### Editing an existing operator index
-An existing operator index can have bundles added to it:
+## Cleaning up e2e tests
+
+#### Delete Machinesets
+
 ```shell script
-opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
+oc delete machineset e2e -n openshift-machine-api
+oc delete machineset e2e-wm -n openshift-machine-api
+oc delete configmap windows-instances -n openshift-windows-machine-config-operator
 ```
-and removed from it:
+
+#### Stash changes
+
+Running e2e tests and deploying the OLM both make changes to the files
+`bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml` and
+`operator-sdk`. Stash changes to those files.
```

Comment:
Why should these changes be stashed? Clean up should revert the change to the bundle. I am not sure what the `operator-sdk` file is.